README.md

```diff
@@ -45,16 +45,46 @@ Git Proxy is built with a developer-first mindset. By presenting simple-to-follo
 
 ## Installation
 
-To install Git Proxy, use the [npm](https://www.npmjs.com/) package manager:
+To install Git Proxy, you must have [Node.js 16 or later](https://nodejs.org/en/download) installed. Use [npm](https://www.npmjs.com) to install the package:
 
 ```bash
-$ npm install @finos/git-proxy
+$ npm install -g @finos/git-proxy
 ```
 
 To install a specific version of Git Proxy, append the version to the end of the `install` command:
 
 ```bash
-$ npm install @finos/git-proxy@1.0.0
+$ npm install -g @finos/git-proxy@1.1.0
+```
+
+To start the server, run `git-proxy`. Alternatively, you can also install & run git-proxy directly using `npx`:
+
+```bash
+$ git-proxy
+# Running with npx - if the package isn't already installed, npx will prompt you to confirm installation
+$ npx --package=@finos/git-proxy@1.1.0 -- git-proxy
+```
+
```

Comment:
@coopernetes - is it worth adding the following here ?

```
### Developement
If you're developing git-proxy and want to run your local checkout, use `npm link`, which will allow to directly invoke the `git-proxy` executable.
```

Feel free to tweak content as you see fit.